atlp-rwanda/rca-barefoot-team6-bn
# #184639058 Project structure with up and running express server

#### What does this PR do?
Creates an express server while also setting up project initial structure
#### Description of Task to be completed?
Has first-end point ``GET /`` and basic folder structure for more work in future.
#### How should this be manually tested?
Close this repository, and install dependencies with ``npm install``, next ``npm run dev`` and check ``http://localhost:3000``
#### Any background context you want to provide?
We are using express server currently, and the only package necessary
#### What are the relevant pivotal tracker stories?
None
